x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/pipelines/ml_inference/add_inference_pipeline_flyout.test.tsx

```diff
@@ -33,7 +33,6 @@ import { NoModelsPanel } from './no_models';
 import { ReviewPipeline } from './review_pipeline';
 import { TestPipeline } from './test_pipeline';
 import { AddInferencePipelineSteps } from './types';
-import { UpdateMappings } from './update_mappings';
```

Comment:
@kderusso Please also delete the component (`update_mappings.tsx`), the unit tests, and the referenced i18n keys from the translation files.

Ideally we should also remove `docLinks.mlDocumentEnrichmentUpdateMappings` which is not used anywhere else in order to not leave it lingering around; it needs to be removed from 4 places if I remember correctly (see [commit that added these](https://github.com/elastic/kibana/commit/8706702aea16ddade385771c71ca3a2a63c45caa)). But if it's too complicated, let's leave it.

Reply:
Done, thanks for the catch!